### PR TITLE
INTYGFV-16625: Added ResponsibleHospName  to CertificateMetadata

### PR DESCRIPTION
**Pull request by mikaelaromild1**

### CODE REVIEW

File: certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/certificate/service/converter/CertificateMetadataConverter.java [hunk 1]
```diff
@@ -114,6 +114,9 @@ public CertificateMetadataDTO convert(Certificate certificate) {
                 .toList()
                 : null
         )
+        .responsibleHospName(certificate.certificateMetaData().responsibleHospName() != null
```
Comment (mhornfeldt) [inline certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/certificate/service/converter/CertificateMetadataConverter.java:117]: Do we want to use "responsibleHospName" or name it something different? I remember we removed the term hosp when working with intyg-proxy.

File: certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/common/ActionEvaluationFactory.java [hunk 2]
```diff
@@ -76,6 +77,7 @@ public ActionEvaluation create(PatientDTO patient, UserDTO user, UnitDTO unit, U
                         .map(HealthCareProfessionalLicence::new)
                         .toList()
                 )
+                .responsibleHospName(new ResponsibleHospName(user.getResponsibleHospName()))
```
Comment (mhornfeldt) [inline certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/common/ActionEvaluationFactory.java:80]: Do we want to use "responsibleHospName" or name it something different? I remember we removed the term hosp when working with intyg-proxy.

**Review by mhornfeldt — APPROVED** [review]

**Review by ChrilleSigerhed — APPROVED** [review]
Looks great! Well done!

image

File: certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/common/ActionEvaluationFactory.java (hunk 2 above)
Comment (ChrilleSigerhed) [reply certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/common/ActionEvaluationFactory.java:80]: Maybe we should update it to ResponsibleIssuer?

**Pull request merged**